doc/CCPPtechnical/source/CompliantPhysicsParams.rst

```diff
@@ -211,8 +274,7 @@ Coding Rules
   or other information needed to initialize the scheme, including stdout and stderr.
   Diagnostic messages are tolerated, but should be minimal.
 
-* Line lengths of up to 120 characters are suggested for better readability (exception: CCPP
-  metadata argument tables).
+* Line lengths of up to 120 characters are suggested for better readability 
```

Comment:
To me, this would be more readable as:
"Line lengths of no more than 120 characters are suggested for better readability"

Reply:
I made this change.